Subject: do not print error on systems without apple motion sensor
From: olh@suse.de

It is not an error if a system has no ams hardware.
Do not clutter dmesg in this case.


---
 drivers/hwmon/ams/ams-core.c |    3 ---
 1 file changed, 3 deletions(-)

Index: b/drivers/hwmon/ams/ams-core.c
===================================================================
--- a/drivers/hwmon/ams/ams-core.c
+++ b/drivers/hwmon/ams/ams-core.c
@@ -219,9 +219,6 @@ int __init ams_init(void)
 		/* Found PMU motion sensor */
 		return ams_pmu_init(np);
 #endif
-
-	printk(KERN_ERR "ams: No motion sensor found.\n");
-
 	return -ENODEV;
 }
 
